### PR TITLE
CLN: docstring cleanup

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/core/indexes/base.py [hunk 1]
```diff
@@ -2412,7 +2412,7 @@ def isna(self) -> np.ndarray:
 
         Returns
         -------
-        numpy.ndarray
+        numpy.ndarray[bool]
```
Comment (jreback) [inline pandas/core/indexes/base.py:2415]: completely minor but i think inconsistent as use of `numpy.ndarray` vs `np.darray` (see this much more) vs `ndarray`

**Review by jreback — APPROVED** [review]
minor comment for followup

File: pandas/core/indexes/base.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/core/indexes/base.py:2415]: i think you're right, decided against trying to make these uniform in this PR

Comment (jbrockmendel) [reply pandas/core/indexes/base.py:2415]: green

Comment (jreback) [reply pandas/core/indexes/base.py:2415]: yep next one (and prob should have a rule for this). note that I am saying it out loud but not knowing how hard that is :->

**Pull request merged**